src/Promitor.Core.Scraping/Configuration/Serialization/Enum/SpecVersion.cs

```diff
@@ -1,8 +1,9 @@
 ﻿namespace Promitor.Core.Scraping.Configuration.Serialization.Enum
 {
-    // ReSharper disable once InconsistentNaming
     public enum SpecVersion
     {
+        // ReSharper disable InconsistentNaming
         v1
+        // ReSharper restore InconsistentNaming
```

Comment:
Yeah - the reason I did this was because originally you'd used DisableOnce, and I had added a `v2` enum item. I switched it like that so that it was ignoring the warning for the shortest possible time, for example:

```csharp
// ReSharper disable InconsistentNaming
v1,
v2
// ReSharper restore InconsistentNaming
```

I can revert the changes if you want, although maybe we're as well leaving it there since we'd just need to add it if we added a `v2` entry?

Reply:
Let's throw it out for now and re-add it when v2 comes back, maybe?

Reply:
Sure - no problem.